packages/@okta/vuepress-theme-prose/components/Home.redesign.vue

```diff
@@ -145,7 +145,7 @@
 
 <script>
 import CompanyLogos from "./CompanyLogos.vue";
-const TABLET_BREAKPOINT = 768;
+const SIDE_BY_SIDE_BREAKPOINT = 1200;
```

Comment:
maybe it will be a good idea to find some more meaningful name

Reply:
Named it as such because that's the "behaviour" were targeting. I know the `1200` breakpoint is `desktop-xl` in `bootstrap. We could use that.

Reply:
I'm ambivalent, but if we go with the bootstrap name, we should put a comment here to explain why we chose the breakpoint (e.g. mention side-by-side behavior)

Reply:
if this variable name describes exactly what it is, and it's obvious for everyone then it's fine to leave the current name. And maybe some extra comment for this breakpoint with some additional information.